Makefile.am

```diff
@@ -148,56 +121,41 @@ libmesos_tests_la_LDFLAGS =				\
   -release $(PACKAGE_VERSION)				\
   -shared $(MESOS_LDFLAGS)
 
-# Test (make check) binary for the dockercfg hook.
-check_PROGRAMS += test-dockercfg
+# Test (make check) binary for the overlay modules.
+check_PROGRAMS = test-overlay
 
-test_dockercfg_SOURCES =				\
-  tests/dockercfg_tests.cpp
+test_overlay_SOURCES =					\
+  tests/overlay_tests.cpp
 
-test_dockercfg_CPPFLAGS =				\
+test_overlay_CPPFLAGS =					\
   $(libmesos_tests_la_CPPFLAGS)
 
-test_dockercfg_LDADD =					\
+test_overlay_LDADD =					\
   $(MESOS_LDFLAGS)					\
   $(MESOS_BUILD_DIR)/$(BUNDLE_SUBDIR)/.libs/libgmock.la	\
   $(MESOS_BUILD_DIR)/src/.libs/libmesos.la		\
   libmesos_tests.la					\
-  libremovedockercfg.la
+  libmesos_network_overlay.la
 
-# Test (make check) binary for the journald module.
-check_PROGRAMS += test-journald
+# Test (make check) binary for the dockercfg hook.
+check_PROGRAMS += test-dockercfg
 
-test_journald_SOURCES =					\
-  tests/journald_tests.cpp
+test_dockercfg_SOURCES =				\
+  tests/dockercfg_tests.cpp
 
-test_journald_CPPFLAGS =				\
+test_dockercfg_CPPFLAGS =				\
   $(libmesos_tests_la_CPPFLAGS)
 
-test_journald_LDADD =					\
+test_dockercfg_LDADD =					\
   $(MESOS_LDFLAGS)					\
   $(MESOS_BUILD_DIR)/$(BUNDLE_SUBDIR)/.libs/libgmock.la	\
   $(MESOS_BUILD_DIR)/src/.libs/libmesos.la		\
   libmesos_tests.la					\
-  libjournaldlogger.la
-
-# Test (make check) binary for the overlay modules.
-check_PROGRAMS += test-overlay
```

Comment:
The diff is a bit misleading, due to the re-rearrangement of the test binaries.  If you un-rearranged them, I think the diff will be much more concise.